configs/katello/4.1.yaml

```diff
@@ -2,17 +2,17 @@
 :project: katello
 :github_org: katello
 :releases:
+  :4.1.1:
+    :redmine_version_id: 1437
   :4.1.0:
     :redmine_version_id: 1351
 :prior_releases:
   :4.0.2:
     :redmine_version_id: 1433
   :4.0.1:
     :redmine_version_id: 1369
-  :3.18.4:
-    :redmine_version_id: 1396
-  :3.18.3:
```

Comment:
3.18.3 is already released, so removing it doesn't help anything now. Better to leave it to indicate that it was really a prior here